.gitignore

```diff
@@ -1,2 +1 @@
 dist/
-.idea/
```

Comment:
I suggest to put this in your global `.gitignore` (see https://github.com/saagie/fluent-bit-mongo/pull/23#discussion_r826147008)